src/main/java/org/acm/auth/commands/PollCommand.java

```diff
@@ -0,0 +1,63 @@
+package org.acm.auth.commands;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+import org.acm.auth.utils.Emoji;
+
+import java.awt.*;
+
+public class PollCommand extends Command{
+
+    // static object for empty perms to be used in the default constructor
+    private static final Permission[] NO_PERMS = {};
+
+    public PollCommand() {
+        super(
+                "poll",
+                "Creates a poll",
+                true,
+                false,
+                new String[] {"vote"},
+                3,
+                20, // Maximum Number of Reactions is 20, and it is kinda overkill to have more than 20 options
+                "-poll Question Option1 Option2 ... OptionN",
+                NO_PERMS,
```

Comment:
The bot needs to have the [MESSAGE_EMBED_LINKS](https://ci.dv8tion.net/job/JDA/javadoc/net/dv8tion/jda/api/Permission.html#MESSAGE_EMBED_LINKS) in order to create embeds. Also, if we want to delete the user's original message after creating the poll, we'd need [MESSAGE_MANAGE](https://ci.dv8tion.net/job/JDA/javadoc/net/dv8tion/jda/api/Permission.html#MESSAGE_MANAGE).